src/ApiFunction.ts

```diff
@@ -23,13 +25,13 @@ export class ApiFunction extends Construct {
   constructor(scope: Construct, id: string, props: ApiFunctionProps) {
     super(scope, id);
     // See https://docs.aws.amazon.com/AmazonCloudWatch/latest/monitoring/Lambda-Insights-extension-versionsx86-64.html
-    const insightsArn = `arn:aws:lambda:${Stack.of(this).region}:580247275435:layer:LambdaInsightsExtension:16`;
-    this.lambda = new Lambda.Function(this, 'lambda', {
-      runtime: Lambda.Runtime.NODEJS_14_X,
+    const insightsArn = `arn:aws:lambda:${Stack.of(this).region}:580247275435:layer:LambdaInsightsExtension:21`;
+    this.lambda = new props.apiFunction(this, 'lambda', {
+      runtime: Lambda.Runtime.NODEJS_18_X,
```

Comment:
Vanochtend met Bram over deze regel gehad. Hier geen node versie opgeven is beter omdat deze overscheven wordt door de versie in de `props.apiFunction` constrcutor. Die versie wordt weer gevuld op basis van de nodejs versie van de projenrc configuratie (in onze globale projen project)